components/SpotlightBanner/__tests__/SpotlightBanner.spec.tsx

```diff
@@ -18,7 +18,7 @@ describe('SpotlightBanner', () => {
   test('renders only SpotlightBannerAside & SpotlightBannerMain', () => {
     const renderer = TestRenderer.create(
       <SpotlightBanner>
-        <SpotlightBannerAside />
+        <SpotlightBannerAside className={'spotlight__comparison'} />
```

Comment:
Undo this please.

Reply:
Create a separate test for testing the `className` prop

Reply:
Updated and pushed